build.cmd

```diff
@@ -0,0 +1,77 @@
+@echo off
+setlocal
+
+:Build
+
+echo **********starting to build the project**********
+set MSBuild="%ProgramFiles(x86)%\MSBuild\12.0\Bin\MSBuild.exe"
+if not exist %MSBuild% @set MSBuild="%ProgramFiles%\MSBuild\12.0\Bin\MSBuild.exe"
+
+set BUILDLOG=BuildResult.txt
+
+:BuildFull
+echo *** Build Microsoft.Odata.Full ***
+
+%MSBuild% sln/Microsoft.OData.Full.sln /m /nr:false  /p:Platform="Any CPU" /p:Desktop=true /v:M /fl /flp:LogFile=bin\msbuild.log;Verbosity=Normal > BUILDLOG
+
+if %ERRORLEVEL% neq 0 goto BuildFail
+goto BuildE2E
+
+
+:BuildE2E
+echo *** Build Microsoft.Odata.E2E ***
+
+%MSBuild% sln/Microsoft.OData.E2E.sln /m /nr:false  /p:Platform="Any CPU" /p:Desktop=true /v:M /fl /flp:LogFile=bin\msbuild.log;Verbosity=Normal >> BUILDLOG

```

Comment:
this would override previous build log file, consider a different log file name?